.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@
 
 TODO:
 
-- [ ] Update CHANGELOG.md with your change (include reference to issue & this PR)
+- [ ] Commits are formatted for [Standard Version](https://github.com/bcoe/conventional-changelog-standard/blob/master/convention.md)
```

Comment:
Hmm, what does the commit format look like? Can we put examples directly in this template?